packages/pages/src/App.js

```diff
@@ -32,7 +33,7 @@ const App = ({ effects, state }) => (
         <input
           type="text"
           id="contentEntry"
-          name="name"
+          name="content"
```

Comment:
Why this change?

Reply:
because the item in question is the "content".

Reply:
Ok, but I don't think we are using these `name` attributes.

If we are not, I think we should remove them.